x-pack/plugins/security_solution/public/management/components/management_empty_state.tsx

```diff
@@ -35,71 +36,124 @@ const PolicyEmptyState = React.memo<{
+                </EuiFlexGroup>
+                <EuiSpacer size="s" />
+                <EuiText size="xs" color="subdued">
+                  <FormattedMessage
+                    id="xpack.securitySolution.endpoint.policyList.onboardingHostInfo"
+                    defaultMessage="Hosts running the Elastic Endpoint"
+                  />
+                </EuiText>
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiFlexGroup>
+                  <EuiFlexItem grow={false} style={{ marginRight: '10px' }}>
+                    <EuiIcon type="tableDensityExpanded" />
+                  </EuiFlexItem>
+                  <EuiFlexItem grow={false} style={{ marginLeft: '0' }}>
+                    <EuiText>
+                      <h4>
+                        <FormattedMessage
+                          id="xpack.securitySolution.endpoint.policyList.onboardingPolicyTitle"
+                          defaultMessage="Policies"
+                        />
+                      </h4>
+                    </EuiText>
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+                <EuiSpacer size="s" />
+                <EuiText size="xs" color="subdued">
+                  <FormattedMessage
+                    id="xpack.securitySolution.endpoint.policyList.onboardingPolicyInfo"
+                    defaultMessage="View and configure protections"
+                  />
+                </EuiText>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+            <EuiSpacer size="xl" />
+            <EuiText size="s" color="subdued">
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.policyList.onboardingSectionThree"
+                defaultMessage="To get started, you’ll have to add the Elastic Endpoint integration to your Agents. Let’s do that now!"
+              />
+            </EuiText>
+            <EuiSpacer size="xl" />
+            <EuiFlexGroup>
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  fill
+                  iconType="plusInCircle"
+                  onClick={onActionClick}
+                  isDisabled={actionDisabled}
+                  data-test-subj="onboardingStartButton"
+                >
+                  <FormattedMessage
+                    id="xpack.securitySolution.endpoint.policyList.actionButtonText"
+                    defaultMessage="Add Endpoint Security"
+                  />
+                </EuiButton>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem>
             <FormattedMessage
-              id="xpack.securitySolution.endpoint.policyList.stepThree"
-              defaultMessage="If you haven’t already, enroll your agents through Fleet using the same agent configuration."
+              id="xpack.securitySolution.endpoint.policyList.placeholdForPictures"
+              defaultMessage="Pictures!!!"
```

Comment:
Suggestion that you use a placeholder pic while the official one is being determined. Maybe:

```
<EuiIcon type="logoSecurity" size="xl" />
```

Reply:
wasn't planning on merging until I got the asset, but that's a good alternative to keep moving